README.md

```diff
@@ -1,16 +1,25 @@
 This is the main development environment for working on Mozilla payments.
 
-TODO: non-development environments are coming soon.
-
 You will need:
 * [Python](https://www.python.org/)
 * [Docker](https://docs.docker.com/)
   * On Mac you can run this with
     [VirtualBox](https://www.virtualbox.org/) and
-    [Boot2Docker](http://boot2docker.io/)
+    [Boot2Docker](http://boot2docker.io/) or
+    [Kitematic](https://kitematic.com/)
```

Comment:
it does actually because it doesn't understand compose natively yet, perhaps i should not add it. its pretty sweet for new people though.